docs/uma-tokenholders/approved-collateral-currencies.md

```diff
@@ -3,81 +3,91 @@ title: Approved collateral currencies
-|MASK| 115 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|BANK| 5.75 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|SFI| 1.3 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|VSP| 50 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|FRAX| 400 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|DEXTF| 2250 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|ORN| 75 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|BOND| 10 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|PUNK-BASIC| 0.015| [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|LUSD| 400 | [UMIP-86](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-86.md)
-|iFARM| 7.5 | [UMIP-88](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-88.md)
-|yvUSDC| 400 | [UMIP-93](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-93.md)
-|UST| 400 | [UMIP-94](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-94.md)
-|BNT| 135 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|vBNT| 400 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|BAND| 75 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|SDT| 450 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|KP3R| 5 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|CREAM| 4 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|CHAIN| 5000 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|ERN| 70 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|OPEN| 1000 | [UMIP-102](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-102.md)
-|DFX| 900 | [UMIP-104](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-104.md)
-|BASK| 1.75 | [UMIP-105](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-105.md)
-|GYSR| 2000 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|MPH| 15 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|APW| 400 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|SNOW| 60 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|NDX| 100 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|Total| No of Approved collateral currencies | [73](https://docs.umaproject.org/uma-tokenholders/approved-collateral-currencies)
+|WETH| 0.2 |  [Ethereum ](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-10.md), [Polygon](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-113.md)
```

Comment:
I don't think this is the best way to organize this. It doesn't make much sense to me to have the network names link to umips. I think a better way would be to have headers of: Collateral Currency | Final Fee | Link to UMIP | Networks Supported

Where there can be multiple link(s) in the Link to UMIP section, and multiple links in the Networks Supported section (these should all be hyperlinks).

Then the links in the Networks Supported section should be to the token contracts on either etherscan or polygon scan.

Wdyt @henrystats ?

Reply:
Updated based on your suggestion. Let me know what you think.

Also did I get BOND's address correctly (Polygon)?

Reply:
Yep BOND looks correct. I left a separate comment on the structure of this table